src/taskEvents/utils.js

```diff
@@ -107,6 +107,99 @@ async function getData(data) {
   };
 }
 
+async function getUserSkills(id) {
+  try {
+    const res = await fetch(`${BASE_URL}/users/${id}/skills`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    const userSkills = await res.json();
+    return userSkills;
+  } catch (error) {
+    alert(error);
```

Comment:
I think it's better to alert a generic message like "Something went wrong" and just console the error for debugging

Reply:
okay got it

Reply:
done